lua/packer.lua

```diff
@@ -663,71 +663,84 @@ end
 
 --- Update the compiled lazy-loader code
 --- Takes an optional argument of a path to which to output the resulting compiled code
-packer.compile = function(raw_args)
+packer.compile = function(raw_args, donot_move_plugins)
```

Comment:
Nitpick: I'd prefer this be `move_plugins` (which will also require changing - but imo simplifying - the conditionals that use this parameter).

Reply:
done